modules/op-release-notes-1-0.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assembly:
+//
+// * openshift_pipelines/op-release-notes.adoc
+
+[id="op-release-notes-1-0_{context}"]
+= Release Notes for Red Hat {pipelines-title} Technology Preview 1.0
+
+[id="new-features-1-0_{context}"]
+== New features
+{pipelines-title} Technology Preview (TP) 1.0 is now available on {product-title} 4.4. {pipelines-title} TP 1.0 is updated to support:
+
+* Tekton Pipelines 0.11.3
+* Tekton `tkn` CLI 0.9.0
+* Tekton Triggers 0.4.0
+* ClusterTasks based on Tekton Catalog 0.11
+
+In addition to the fixes and stability improvements, here is a highlight of what’s new in OpenShift Pipelines 1.0.
+
+.Pipelines
+* Support for  v1beta1 API Version.
+* Support for an improved LimitRange. `limitRangeName` is now removed, and TaskRun now accounts for LimitRange on behalf of the user.
+* Support for sharing data between Tasks using Task Results and Task Params.
+* Pipelines can now be configured to not overwrite the `HOME` environment variable and `workingDir` of Steps.
+* Similar to Task Steps, `sidecars` now support script mode.
+* You can now specify a different scheduler name in TaskRun `podTemplate`.
+* Support for variable substitution using Star Array Notation
+* Tekton Controller can now be configured to monitor an individual namespace.
+* A new description field is now added to the specification of Pipeline, Task, ClusterTask, Resource, and Condition.
+* Addition of  proxy parameters to Git PipelineResources.
+
+.`tkn` CLI
+
+* The `describe` sub-command is now added for the following `tkn` resources -- `eventlistener`, `condition`, `triggertemplate`, `clustertask`, and `triggerbinding`.
```

Comment:
Also `clustertriggerbinding`

Reply:
@savitaashture - for ClustertriggerBinding a new entry is added later 
* A new `clustertriggerbinding` command is now added with sub-commands - `describe`, `delete`, and `list`.